### PR TITLE
Add twig.runtime

### PR DESCRIPTION
**Pull request by ismail1432**
Hi everyone, 

The  `twig.runtime` is missing in the DIC list, [see](https://github.com/symfony/symfony-docs/issues/10192)
Maybe there is a bunch of errors as It's the first time I'm writing a complete paragraph,
As usual Reviews are welcome !

### CODE REVIEW

File: reference/dic_tags.rst [hunk 1]
```diff
@@ -1175,6 +1176,55 @@ also register it manually:
...
+        use App\Twig\AnotherExtension;
+
+        $container
+            ->register(AppExtension::class)
+            ->addTag('twig.runtime')
+        ;
+       
+For information on how to create the actual Runtime Twig Extension class, see
+`Twig's documentation`_ on the topic or read the
+:doc:`/templating/twig_extension#creating-lazy-loaded-twig-extensions` article.
```
Comment (xabbuh) [inline reference/dic_tags.rst]: If we want to link to a particular section, we will have to use the `ref` role like this:

```rst
:ref:`Creating Lazy-Loaded Twig Extensions <lazy-loaded-twig-extensions>`
```

and the headline in `templating/twig_extension.rst` must be prefix now with the desired label:

```rst
.. _lazy-loaded-twig-extensions:

Creating Lazy-Loaded Twig Extensions
------------------------------------
```

**Comment by weaverryan** [comment]
Hey @ismail1432!

This is actually beautiful, nice job! However... we might now have some duplication :). We talk about this topic in [templating/twig_extension.rst](https://github.com/symfony/symfony-docs/blob/master/templating/twig_extension.rst#creating-lazy-loaded-twig-extensions) and the `twig.runtime` tag is autoconfigured. We definitely *should* add this tag to this page. But, I think we should mostly just link to that other page. If there are some improvements to that other location that you can make based on your docs, that would be even better :).

Status: Needs work

**Comment by javiereguiluz** [comment]
@weaverryan I thought there was some duplication also ... but in the templating/twig_extension.rst page we no longer display how to register this tag manually, so we could keep it in reference/dic_tags.rst as proposed by @ismail1432.

**Comment by javiereguiluz** [comment]
@ismail1432 thanks for this contribution! We finally merged it.

**Pull request closed without merging**